File: bmi/bmimetric.py
# For example, an execution could look like this:

#    Please enter weight in kilograms: 50
#    Please enter height in meters: 1.58
#    BMI is: 20.0288415318

weight = float(input("Please enter weight in kilograms: "))
height = float(input("Please enter height in meters: "))
print(f"BMI is: {weight / height / height}")

File: bmi/bmimetric.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
    int weight;
    float height;
    cout << "Please enter weight in kilograms: ";
    cin >> weight;
    cout << "Please enter height in meters: ";
    cin >> height;
    cout << "BMI is: " << ceil(weight / (height * height) * 100.0) / 100.0 << endl;
    return 0;
}